### PR TITLE
Modified for i18n

### PR DESCRIPTION
**Pull request by jkofsky**
Last post.

### CODE REVIEW

**Review by bluezed — CHANGES_REQUESTED** [review]
OK... I'll merge it in if you revert back to the original circle button design. Your new version does not work with my own layouts as it takes up too much space.

Also please remove the whole $linkText part. It's just not something I feel is very useful and it's causing more issues than it addresses. eg. the font for the text would have to be matched to whatever the application uses and also the formatting looks a bit off when I just tried it.
Also the 'title' causes problems in my applications as I have certain styles for my tooltips and they get "fed" from the titles so I'd have to add styling in here too.

Basically any update should preserve how the widget worked before and any new options should be "opt-in" only.

Thanks!
